### PR TITLE
Build a tool bar(or header) and headline section

### PR DESCRIPTION
**Pull request by d4nQw3rty**
*Build a tool bar
*Build headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,40 @@
...
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindy ensure your logo is wrapped in an `a` tag, this is for the user experience as the ogo acts as a button to take the users to the homepage.

File: index.html [hunk 2]
```diff
@@ -5,9 +5,40 @@
...
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly ensure your hamburger menu is wrapped in a `nav` tag,  this tag is for semantics reasons and also helpful when positioning elements in mobile view.

File: index.html [hunk 3]
```diff
@@ -5,9 +5,40 @@
...
+          </div>
+          <ul id="social-media-icons">
+            <li><a href="https://twitter.com/Danys_Linares" target="_blank" rel="noopener noreferer"><img src="img/Vector-twitter.svg" alt="Twitter link"></a></li>
+            <li><a href="https://www.linkedin.com/in/danys-linares-6a328b238?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BnkyI5IMjTzSg4PVJIZh%2BMw%3D%3D" rel="noopener noreferer"><img src="img/Linkedin-icon.svg" alt="Linkedin link"></a></li>
+            <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+            <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+            <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+         </ul>                                  
+      </div>
+    </div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly change your `div` tag to a `section` tag wrapping the whole section, the element represents a generic standalone section of a document, learn more[ here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

File: index.html [hunk 4]
```diff
@@ -5,9 +5,40 @@
...
+             <p id="My-Logo">Qwerty</p>
+         </div>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
+      </header>
+      <div id="Headline">
+         <h2 id="Primary-Text">I’m Danys
+             Glad to see you!
+         </h2>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly change your header tag to h1 since this is your first header tag, this will make your page optimized for SEO.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @d4nQw3rty , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] It would be nice adding more sections to your readme file eg getting started section, also your MIT license needs to be personalized

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 5]
```diff
@@ -14,6 +14,12 @@ build the first 2 sections of the mobile website using the template you chose in
 - git and gitHub
 - Visual Code
```
Comment (prolajumokeoni) [inline README.md:17]: ### OPTIONAL

-  Well done making your readme file descriptive, kindly ensure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) are present 👍🏿

File: index.html [hunk 6]
```diff
@@ -5,9 +5,40 @@
...
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <nav id="Logo-Placeholder">
+             <a href="#" id="My-Logo"><img src="img/My_logo.png" alt="Home page"></a>
+         </nav>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
```
Comment (prolajumokeoni) [inline index.html]: `<nav id="Normal-Button-Tertiary-Enabled">         
            <div id="icon-menu">Menu</div>
         </nav>`

- [ ] Just as the previous reviewer requested, wrap the hamburger menu icon in a nav tag, currently you wrapped the logo in the nav tag and not the menu icon, please change this 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @d4nQw3rty, 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @d4nQw3rty 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**